configuration/w3m.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: doc
+title: Reducing the fingerprint of the text-based web browser w3m
+permalink: /doc/w3m/
+redirect_from:
+- /en/doc/mutt/
+- /doc/W3m/
+- /wiki/W3m/
+---
+
+Reducing the fingerprint of the text-based web browser w3m
+====
+
+[w3m](http://w3m.sourceforge.net/) 'is a text-based web browser as well as a pager like `more` or `less`. With w3m you can browse web pages through a terminal emulator window (xterm, rxvt or something like that). Moreover, w3m can be used as a text formatting tool which typesets HTML into plain text.'
+
+You can reduce the [fingerprint](https://panopticlick.eff.org/about#browser-fingerprinting) of w3m by adjusting some settings to those of the Tor Browser Bunde (TBB) with JavaScript disabled.
+
+**BEWARE: As very few people use w3m for browsing chances are high that you will still be the only person with this fingerprint on your adversary's radar. Also, I am nothing but a wannabe security expert, so do not rely on my advise for anything critical.**
+
+Apply the following changes to `~/.w3m/config` in any AppVM you want to use w3m in. If you have not run w3m yet, you might need to copy the config file from elsewhere. You can also apply the same changes to `/etc/w3m/config` in the relevant TempVM(s) to have them apply to multiple AppVMs; but make sure they are not reversed by the contents of `~/.w3m/config` in any of the AppVMs. (w3m reads `~/.w3m/config` after `/etc/w3m/config`).
```

Comment:
We don't use `TempVM` term, use `TemplateVM` instead, see https://www.qubes-os.org/doc/glossary/